modules/openapi-generator/src/test/java/org/openapitools/codegen/java/jaxrs/JavaJAXRSSpecServerCodegenTest.java

```diff
@@ -0,0 +1,109 @@
+package org.openapitools.codegen.java.jaxrs;
+
+import io.swagger.v3.oas.models.Operation;
+import org.junit.Before;
+import org.junit.Test;
+import org.openapitools.codegen.CodegenOperation;
+import org.openapitools.codegen.languages.JavaJAXRSSpecServerCodegen;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Unit-Test for {@link org.openapitools.codegen.languages.JavaJAXRSSpecServerCodegen}.
+ *
+ * @author attrobit
+ */
+public class JavaJAXRSSpecServerCodegenTest {
+    
+    private JavaJAXRSSpecServerCodegen instance;
+    
+    @Before
+    public void before() {
+        instance = new JavaJAXRSSpecServerCodegen();
+    }
+    
```

Comment:
You could remove a line here

Reply:
done.

Reply:
@JFCote Whats next? I cant merge the PR. Do you merge the PR?